lib/api/route/messages.dart

```diff
@@ -262,9 +262,10 @@ Future<UploadFileResult> uploadFile(
   required Stream<List<int>> content,
   required int length,
   required String filename,
+  required String? mimeType,
```

Comment:
Probably cleanest to call the parameter `contentType`.

In general we try to make our names match the names in the API. Here this doesn't correspond to anything mentioned in the API docs:
https://zulip.com/api/upload-file

but it does correspond to a `content-type:` header in the request. So that seems like the canonical name to use.